__tests__/app.test.js

```diff
@@ -203,3 +203,56 @@ test("400 should respond with an error message for an invalid ID endpoint", () =
       expect(body).toEqual({ msg: "Bad request" });
     });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("201: should add a comment for a valid article ID", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "The cutest man in all of the land!",
+    };
+
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment).toMatchObject(  {
+          comment_id: 19,
+          body: 'The cutest man in all of the land!',
+          article_id: 1,
+          author: 'butter_bridge',
+          votes: 0,
+          created_at: expect.any(String),
+        })
+      });
+  });
+  test("404: should respond with an error message for a non-existent article ID", () => {
+    const newComment = {
+      username: "Eric Cantona",
```

Comment:
I'd seperate out a test for the none existent id vs using a username that does not exist as these are seperate functionalitys and we only really want to change one variation at a time.